tiledb/sm/query/ast/query_ast.cc

```diff
@@ -203,14 +203,14 @@ void ASTNodeExpr::get_field_names(
 
 bool ASTNodeExpr::is_or_supported() const {
   if (combination_op_ != QueryConditionCombinationOp::AND) {
-    return false;
+    return true;
```

Comment:
Why is it `!= QueryConditionCombinationOp::AND` and not `== QueryConditionCombinationOp::OR`?

Reply:
i think this is resolved with the rename.